packages/remix-dev/config/defaults/entry.server.cloudflare.tsx

```diff
@@ -13,7 +13,9 @@ export default async function handleRequest(
   const body = await renderToReadableStream(
     <RemixServer context={remixContext} url={request.url} />,
     {
-      signal: request.signal,
+      // If you wish to abort the rendering process, you can pass a signal here.
+      // Please refer to the templates for example son how to configure this.
+      // signal: controller.signal,
```

Comment:
Stop passing the `request.signal` to fix the issue - but do not add any default timeout behavior to avoid a breaking change